src/main.rs

```diff
@@ -12,6 +12,7 @@ use crossterm::{
 		enable_raw_mode, window_size
 	}
 };
+use csscolorparser;
```

Comment:
```suggestion
```
Something similar to this used to be neccessary before rust 2018, but you don't need to explicitly import a crate into a namespace to use it anymore